### ISSUE TITLE
Evaluate the effect of image transforms on model performance

### ISSUE THREAD

**Issue by havanagrawal** [havanagrawal/clomask#20]
As discussed in #16 it would be interesting to check if the brightness, clarity, and perspective have any effect on the model performance. My personal intuition is that:
1. Brightness and perspective are less likely to affect the model performance significantly
2. The clarity (blurring/low-pass filters) will have some effect on the model performance.

**Deliverables**

1. A Jupyter notebook with the above analysis
2. A README document with appropriate details

**Comment by havanagrawal** [comment]
@lmtoan I have a question regarding this issue. 

What benefit do you see us receiving out of the results? Even if we do determine that certain transformations affect the model performance, these will be specific to the image at hand. I don't believe that deep learning models expect the input image to be specifically processed in any way (except being cropped/have a certain resolution sometimes).

**Comment by lmtoan** [comment]
Thanks @havanagrawal. What you said makes sense. Artificially transforming and distorting images might not change the model performance much.

My initial suggestion was to further investigate the model robustness with regards to bottles. We already tested the COCO model on image resolution, image content alignment (slanted bottles), but not yet on image quality. From what I remembered, Clobotics was struggling with bad lighting and blurred images from clients so I thought this might appeal to them.

We can close this issue or if you have other suggestions on testing the model with regards to different image qualities. Thanks!
